diff --git a/node_modules/ansi-html/index.js b/node_modules/ansi-html/index.js
index 7c942d6..49df1eb 100644
--- a/node_modules/ansi-html/index.js
+++ b/node_modules/ansi-html/index.js
@@ -6,7 +6,7 @@ module.exports = ansiHTML
 var _regANSI = /(?:(?:\u001b\[)|\u009b)(?:(?:[0-9]{1,3})?(?:(?:;[0-9]{0,3})*)?[A-M|f-m])|\u001b[A-M]/
 
 var _defColors = {
-  reset: ['fff', '000'], // [FOREGROUD_COLOR, BACKGROUND_COLOR]
+  reset: ['fff', ''], // [FOREGROUD_COLOR, BACKGROUND_COLOR]
   black: '000',
   red: 'ff0000',
   green: '209805',
